### PR TITLE
[WIP] Added multi-organism support for Apollo-report

### PR DESCRIPTION
**Pull request by mboudet**

### CODE REVIEW

File: apollo_checker/report.sh [hunk 1]
```diff
@@ -40,23 +28,51 @@ then
...
+    echo -n "\"$orga\""":" >> "$output_dir/full_report.json"
+    cat "$orga_output_dir/report.json" >> "$output_dir/full_report.json"
+    echo "" >> "$output_dir/full_report.json"
+    cp "$raw_apollo_gff" "$orga_output_dir/$raw_apollo_gff"
+
+    # Remove temp files
+    rm -rf $tmp_dir
+done
+
+sed -i '1s/^/{/;$!s/$/,/;$s/$/}/' "$output_dir/full_report.json"
```
Comment (abretaud) [inline apollo_checker/report.sh:78]: what's this?

File: apollo_checker/report_local.sh [hunk 2]
```diff
@@ -12,10 +12,13 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Automatic config
 tmp_dir=`mktemp -d`
 raw_apollo_gff="/data/raw_apollo.gff"
+organism_name="Genus Species"
 cur_date=`date +%F_%X`
 
 cd "$tmp_dir"
 
+echo -n "" > "$output_dir/full_report.json"
```
Comment (abretaud) [inline apollo_checker/report_local.sh:20]: and this?

File: apollo_checker/report_local.sh [hunk 3]
```diff
@@ -12,10 +12,13 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Automatic config
 tmp_dir=`mktemp -d`
 raw_apollo_gff="/data/raw_apollo.gff"
+organism_name="Genus Species"
```
Comment (abretaud) [inline apollo_checker/report_local.sh]: Maybe "Unknown organism"?

**Comment by abretaud** [comment]
Looks good thanks!
I'm tempted to just merge and see if it works live, I have an apollo ready for that

File: apollo_checker/report.sh (hunk 1 above)
Comment (mboudet) [reply apollo_checker/report.sh:78]: It basically transform a multiline file in proper json : wrap everythin in {}, and add "," after each line except last.

File: apollo_checker/report_local.sh (hunk 2 above)
Comment (mboudet) [reply apollo_checker/report_local.sh:20]: Create the file if not existing, empty it if it exist (without newline for the regex)

File: apollo_checker/report_local.sh (hunk 3 above)
Comment (mboudet) [reply apollo_checker/report_local.sh]: Sure

**Comment by abretaud** [comment]
allez, zou!

**Comment by abretaud** [comment]
oh damnèd...
I forgot that there is one genome fasta file per organism. So the genome var in https://github.com/abretaud/docker-apollo-report/blob/master/apollo_checker/report.sh#L9 makes no sense now, it should change for each organism

**Comment by abretaud** [comment]
(+gpg error in https://quay.io/repository/abretaud/apollo-report/build/e87afd50-1f8f-41b4-b29d-b0b1d09106d5, I'll fix it)

**Pull request merged**